async-query-core/src/main/java/org/opensearch/sql/spark/client/EMRServerlessClientFactoryImpl.java

```diff
@@ -27,13 +26,8 @@ public class EMRServerlessClientFactoryImpl implements EMRServerlessClientFactor
   private EMRServerlessClient emrServerlessClient;
   private String region;
 
-  /**
-   * Gets an instance of {@link EMRServerlessClient}.
-   *
-   * @return An {@link EMRServerlessClient} instance.
-   */
   @Override
-  public EMRServerlessClient getClient() {
+  public EMRServerlessClient getClient(String accountId) {
```

Comment:
Where are we using the accountId? would it only be used in managed service?

Reply:
// TODO: It does not handle accountId for now. (it creates client for same account)
There is this TODO in this file, is this related?

Reply:
Also cluster name should include accountID for managed service

Reply:
Yes, it would be used only for managed service. The TODO is mentioning different thing. I think the intention of TODO is to make it possible to use EMRS cluster in different account.
The accountId passed here is requester's accountId.

> @noCharger
What do you mean?

Reply:
@ykmr1224  https://github.com/opensearch-project/sql/blob/00f82f52dfca873f083ce0b7791a2b69bba0175c/async-query-core/src/main/java/org/opensearch/sql/spark/asyncquery/model/SparkSubmitParameters.java#L110-L111
clustername is in format of <accountid>:<domain name>